### DISCUSSION TITLE
Search with complex searchTerm not working

### DISCUSSION THREAD

**Discussion by ThomekSolutions** [mozts2005/ZendeskApi_v2#508] [Q&A] [answered]
Hello,

I have a small issue with the search.

When i try this query : https://XXX.zendesk.com/api/v2/search.json?query=created%3E2021-03-17T14:39:03%2B00:00%20created%3C=2021-03-17T14:39:33%2B00:00%20status:open%20type:ticket%20tags:intervention

The result is fine.

But if y try to use this query in the search code, no result found :

```
var query = "created%3E2021-03-17T14:39:03%2B00:00%20created%3C=2021-03-17T14:39:33%2B00:00%20status:open%20type:ticket%20tags:intervention";
_api.Search.SearchFor<Ticket>(query);
```

Any Idea please ?

Thanks in advance

**Comment by mozts2005** [comment, marked as answer]
@tpettinicchi It looks like your query is URL encoded. The library does that for you so the URL will be double encoded.
